src/module.ts

```diff
@@ -152,14 +152,14 @@ export default defineNuxtModule<ModuleOptions>({
         _tailwindConfig.content = _tailwindConfig.purge
```

Comment:
I also want to propose now removing consideration of `purge`

Reply:
Might be a breaking change though

Reply:
The module uses Tailwind CSS 3 which mentions in migration guide to use `content` rather than `purge`: https://tailwindcss.com/docs/upgrade-guide#configure-content-sources